config/routes.rb

```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  
+  root 'home#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.